src/main/java/org/apache/commons/io/build/AbstractOrigin.java

```diff
@@ -227,7 +476,7 @@ public CharSequence getCharSequence(final Charset charset) {
         @Override
         public InputStream getInputStream(final OpenOption... options) throws IOException {
             // TODO Pass in a Charset? Consider if call sites actually need this.
-            return CharSequenceInputStream.builder().setCharSequence(getCharSequence(Charset.defaultCharset())).get();
+            return CharSequenceInputStream.builder().setCharSequence(getCharSequence(null)).get();
```

Comment:
How is this change related to Channels? This is a giant PR already. Can it be focused on just the new feature? Or is this a bug fix?

Reply:
Sorry, fixed in https://github.com/apache/commons-io/pull/784/commits/24fff74dc1645e7b566fb167befe1ad167c1ba62